src/Features/Core/Portable/NavigateTo/NavigateToSearcher.cs

```diff
@@ -19,6 +20,14 @@
 
 namespace Microsoft.CodeAnalysis.NavigateTo
 {
+    [Flags]
+    internal enum NavigateToSearchScope
+    {
+        RegularDocuments = 0b01,
```

Comment:
where do the enum values come from?

Reply:
Not sure what you mean.  Can you clarify?

Reply:
Ah - I meant why the value `0b01`?

Reply:
Ah. It's a flags enum. So I prefer using binary to show the values and how they can be combined.